ansible/playbook_rollout.yml

```
@@ -44,3 +44,4 @@
     - role: mongodb_query_exporter
     - role: ingress
     - role: post_deployment
+    - role: tldraw-client-core
```

the tldraw-client-core ansible role must bevor the post_deployment role
```suggestion
    - role: mongodb_query_exporter
    - role: tldraw-client-core
    - role: ingress
    - role: post_deployment
```